src/Components/CreateImageWizard/steps/Packages/PackageRecommendations.tsx

```diff
@@ -210,8 +210,10 @@ const PackageRecommendations = () => {
               <>
                 <Content>
                   <Content>
-                    Other users commonly add these packages with the ones you
-                    selected.
+                    Dependencies for your selected packages will be
+                    automatically included in the blueprint. The recommended
+                    packages are optional suggestions based on what other users
+                    add in similar setups.
```

Comment:
I suggest turning these statements around. First explain what the feature does, then specify that they are not dependencies.